### PR TITLE
Replace certificate language in lms and core

### PR DESCRIPTION
**Pull request by mumarkhan999**
#### Story Link
[https://edlyio.atlassian.net/browse/EDS-245](https://edlyio.atlassian.net/browse/EDS-245)

#### PR Description

The language of "certificate" has been replaced with "statement of accomplishment"
#### Type of change

Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change
- [X] Content change

#### Checklist before merging:

- [ ] Squased
- [ ] Reviewd

### CODE REVIEW

File: common/static/common/js/components/PortfolioExperimentUpsellModal.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export class PortfolioExperimentUpsellModal extends React.Component {
                 <img
                     className="upsell-certificate"
                     src="https://courses.edx.org/static/images/edx-verified-mini-cert.png"
-                    alt="Sample verified certificate"
+                    alt="Sample verified statement of accomplishment"
```
Comment (tasawernawaz) [inline common/static/common/js/components/PortfolioExperimentUpsellModal.jsx]: remove verified from here.

Comment (mumarkhan999) [reply common/static/common/js/components/PortfolioExperimentUpsellModal.jsx]: done.

File: lms/djangoapps/certificates/views/webview.py [hunk 2]
```diff
@@ -72,13 +72,14 @@ def get_certificate_description(mode, certificate_type, platform_name):
     elif mode == 'verified':
         # Translators:  This text describes the 'ID Verified' course certificate type, which is a higher level of
         # verification offered by edX.  This type of verification is useful for professional education/certifications
-        certificate_type_description = _("A {cert_type} certificate signifies that a "
+        # ucsd-customizations: Here I am removing {cert_type} as ucsd is only using verified certificates so we
```
Comment (HamzaIbnFarooq) [inline lms/djangoapps/certificates/views/webview.py]:
```suggestion
        # [UCSD_CUSTOM] [EDS-SOMETHING]: Here I am removing {cert_type} as ucsd is only using verified certificates so we
```
can we do something like this for consistency?

Comment (mumarkhan999) [reply lms/djangoapps/certificates/views/webview.py]: done.

**Pull request merged**